iris-mpc-cpu/src/protocol/binary.rs

```diff
@@ -305,14 +299,17 @@ where
     let (m0, m1, wc) = {
         let reply_m0_and_m1 = network.receive_next().await;
         let m0_and_m1 = NetworkValue::vec_from_network(reply_m0_and_m1).unwrap();
-        assert!(
-            m0_and_m1.len() == 2,
-            "Deserialized vec in bit inject is wrong length"
-        );
-        let (m0, m1) = m0_and_m1.into_iter().collect_tuple().unwrap();
-
-        let m0 = T::into_vec(m0);
-        let m1 = T::into_vec(m1);
+        if m0_and_m1.len() == 2 {
```

Comment:
this seems wrong way round at first glance?

Reply:
Yep, I already fixed it.

Reply:
my bad, apparently had review open on outdated version